src/test/common/TestdataGenerator.kt

```diff
@@ -1616,3 +1618,33 @@ fun lagRelatertBehandling(
         vedtattTidspunkt = vedtattTidspunkt,
         fagsystem = fagsystem,
     )
+
+fun lagBarnehageBarn(
```

Comment:
Liten typo, greit å fikse så det er likt med typen som returneres

```suggestion
fun lagBarnehagebarn(
```